eigen_pod_proofs.go

```diff
@@ -39,7 +40,7 @@ func NewEigenPodProofs(chainID uint64, oracleStateCacheExpirySeconds int) (*Eige
 	}, nil
 }
 
-func (epp *EigenPodProofs) ComputeBeaconStateRoot(beaconState *capella.BeaconState) (phase0.Root, error) {
+func (epp *EigenPodProofs) ComputeBeaconStateRoot(beaconState *deneb.BeaconState) (phase0.Root, error) {
```

Comment:
can we just define new functions on `*deneb.BeaconState`? like

```go
func (s *deneb.BeaconState) ComputeBeaconStateRoot() (phase0.Root, error) {
```

not sure if go allows it because its from another module